### DISCUSSION TITLE
GSoC 2023 Application Window is Open

### DISCUSSION THREAD

**Discussion by mshudrak** [google/gcp_scanner#112] [Announcements]
Hi everyone,

As some of you might know, GSoC 2023 opened application process for candidates. The window will remain open until April 4. Please note that we will be accepting applications submitted only through [GSoC 2023 official portal](https://summerofcode.withgoogle.com/). We will not consider applications submitted to the GCP Scanner Google Group or anywhere else including mentors personal email addresses.

If you want any feedback on your proposal, please share it with gcp_scanner@google.com and send an email to the same address expressing your interest in receiving feedback. Please be patient since we are experiencing high interest in the project and we have limited resources.

Please refrain from using generative AI tools to write your proposals, they usually don't match what we expect. We encourage you to submit your own original ideas to the project.

**Comment by mshudrak** [comment]
You can use generative AI when you need to check the language or improve your sentences but applications fully written by generative AI does not really demonstrate that you can independently come up with ideas for improving the project.

**Comment by tausiq2003** [comment]
Makshim, what should I write in Bibliography and Link to prototypes?

**Comment by mshudrak** [comment]
You can add links to any work you've previously done in open-source/research space e.g. research papers, blogposts, PRs and etc.

**Comment by tausiq2003** [comment]
And in bibliography?

**Comment by mshudrak** [comment]
Research papers you've done.

**Comment by tausiq2003** [comment]
Hey @mshudrak can you please look at my comment replies in my proposal.

**Comment by tausiq2003** [comment]
The proposal is not following [this]( https://google.github.io/gsocguides/student/writing-a-proposal)  should I have rewrite or its right?

**Reply by tausiq2003** [reply]
> The proposal is not following [this](https://google.github.io/gsocguides/student/writing-a-proposal) should I have rewrite or its right?

Make a final review and suggest me what to improve, I think you have got my reply on google docs, also please answer to the above question. @mshudrak

**Reply by mshudrak** [reply]
@tausiq2003 it is ok. Our proposal template mostly covers everything discussed in the doc you linked.

**Reply by tausiq2003** [reply]
thanks
